modules/sdk-unified-wallet/src/ecdsa-unified-wallet/types.ts

```diff
@@ -1,4 +1,4 @@
 import { CoinFamily } from '@bitgo/statics';
 
-export const supportedCoins: CoinFamily[] = [CoinFamily.ETH, CoinFamily.BSC, CoinFamily.POLYGON];
+export const supportedCoins: string[] = [CoinFamily.ETH, CoinFamily.BSC, CoinFamily.POLYGON];
```

Comment:
why relax the type here?

Reply:
instead of just changing this to `string` and add extra param in generateUnifiedWallet
Can we just add test coins in the list?

Reply:
Or if it's only for testing purpose, we could hard modify in the test, but i really dont think we should add a param just for test